data/benefits/javascript/region-hauts-de-france-fonds-regional-social-urgence.yml

```diff
@@ -1,9 +1,9 @@
 label: fonds Régional Social d’Urgence (FRSU)
 institution: hauts_de_france
 description: La Région propose d'apporter une aide financière personnalisée aux
-  étudiant·e·s en difficulté financière.
+  étudiants ou étudiantes en difficulté financière.
```

Comment:
remarque générale, que pensez vous d'écrire plutôt "étudiants ET étudiantes" ? food for thoughts